source/installation/packages.rst

```diff
@@ -21,12 +21,17 @@ can help with simple config questions, for anything else we concentrate on
 current versions.
 
 The rsyslog project offers current packages for a number of major distributions.
-More information about these can be found at http://www.rsyslog.com/downloads/download-other/.
+More information about these can be found at the |RsyslogPackageDownloads|_
```

Comment:
This actually creates a hyperlink with the text of "rsyslog package downloads" that points to the original URL. I think this is actually a good example of the approach that I think you were recommending that we use.